css/global.css

```diff
@@ -30,17 +30,82 @@ Variables (colors, text, spacing, etc.)
 :root {
 	/* Color Definitions */
 	--color-white: #fff;
+	--color-amaranth: #b91515;
+	--color-teal: #007a80;
+	--color-cybergrape: #6c4681;
+	--color-onyx: #1f1d1d;
+	--color-yaleblue: #184f85;
+	--color-hotpink: #fa6cb7;
+	--color-citrine: #d3c53f;
+	--color-citron: #92a32d;
+	--color-copper: #b37122;
+	--color-bluepurple: linear-gradient(180deg, #6c4681 -5.71%, #184f85 103.44%);
+	--color-yellowgreen: linear-gradient(
+		180deg,
+		#d3c53f 0%,
+		#d3c53f 26.04%,
+		#92a32d 58.85%,
+		#b37122 100%
+	);
 
 	/* Color Assignment */
 	--body-background-color: var(--color-white);
+	--button1-color: var(--color-white);
+	--button1-hover-border: var(--color-white);
+	--button2-color: var(--color-amaranth);
+	--button2-hover: var(--color-white);
+	--button3-color: var(--color-amaranth);
+	--button3-hover-border: var(--color-amaranth);
 
 	/* Text */
+	--headings-text: "Josefin Sans", sans-serif;
+	--body-text: "Josefin Sans", sans-serif;
+	--button-text: "Josefin Sans", sans-serif;
+
```

Comment:
add some comments for separation

Reply:
looks good! thanks for keeping vars DRY

Reply:
Thank you kindly! comments were separated!